polling_stations/apps/data_importers/management/commands/run_new_imports.py

```diff
@@ -0,0 +1,167 @@
+            help="<Optional> The earliest commit hash against which to search for changed scripts. "
+            "Defaults to value of LAST_IMPORT_SHA in ssm parameter store",
+        )
+        parser.add_argument(
+            "-t",
+            "--to-sha",
+            help="<Optional> The latest commit hash against which to search for changed scripts. Defaults to HEAD",
+            default=git_rev_parse("HEAD"),
+        )
+        parser.add_argument("--post-deploy", action="store_true", default=False)
+
+    def run_scripts(self, changed_paths, opts):
+        for script in changed_paths:
+            script_path = Path(script)
+            try:
+                call_command(script_path.stem, **opts)
+                self.summary.append(
+                    (
+                        "INFO",
+                        f"Ran import script: {script_path.name}",
+                    )
+                )
+            except:
+                # usually we want to handle a specific exception, but in this situation
+                # if there is any issue (at all) trying to run the command,
+                # we just want to log it and move on to the next script
+                self.summary.append(
+                    ("WARNING", f"{script_path.name} could not be run!")
+                )
+                continue
+
+    def output_summary(self):
+        for line in self.summary:
+            if line[0] == "INFO":
+                self.stdout.write(line[1])
+            elif line[0] == "WARNING":
+                self.stdout.write(self.style.ERROR(line[1]))
+            else:
+                self.stdout.write(line[1])
+
+    def handle(self, *args, **options):
+        self.check(
+            [
+                apps.get_app_config("data_importers"),
+                apps.get_app_config("pollingstations"),
+            ]
+        )
+
+        if not (from_sha := options.get("from_sha")):
+            from_sha = get_last_import_sha_from_ssm()
+
+        to_sha = options.get("to_sha")
+        is_post_deploy = options.get("post_deploy")
+
+        changed_paths = get_paths_changed(from_sha, to_sha)
+        changed_scripts = get_changed_scripts(changed_paths)
+        has_imports = any_import_scripts(changed_paths)
+        has_application = any_non_import_scripts(changed_paths)
+
+        cmd_opts = {
+            "nochecks": True,
+            "verbosity": 1,
+            "use_postcode_centroids": False,
+            "include_past_elections": False,
+        }
+
+        sys.stdout.write(
+            f"Comparing repo between {from_sha} and {to_sha}\n"
+            f"\tFrom: https://github.com/DemocracyClub/UK-Polling-Stations/commit/{git_rev_parse(from_sha)}\n"
+            f"\tTo: https://github.com/DemocracyClub/UK-Polling-Stations/commit/{git_rev_parse(from_sha)}\n"
+        )
+
+        # __import__("ipdb").set_trace()
+        if has_imports and not has_application:
+            sys.stdout.write("Only import scripts have changed\n")
```

Comment:
I think you want `self.stdout` in these. `sys.stdout.write` is the same as `print`, but it's useful to use whatever `self.stdout` is set to in management commands

Reply:
rebase fail - I did that here 83ccb291a637243967da56b12e27ce37f7e4a667